### ISSUE TITLE
Installer of the standalone version under Windows get a CRC Error

### ISSUE THREAD

**Issue by Serverfrog** [aptana/studio3#359]
![image](https://cloud.githubusercontent.com/assets/1306404/11967892/f85fbdb2-a906-11e5-9e9a-da7fe0701b75.png)

**Comment by Serverfrog** [comment]
already Posted on https://jira.appcelerator.org/browse/APSTUD-8446 on the 1.12.2015. Did you even work on this?

**Comment by ingo** [comment]
Different cause. To fix (for now) install Node 10.X manually and then restart the installer.

**Comment by Serverfrog** [comment]
nodejs 10? they are at 5

**Comment by ingo** [comment]
Yes, 0.10.x. Apologies

**Comment by millsy123** [comment]
could you please provide the nodejs 0.10.x link to download? 

I have installed in my system but still it is showing the same CRC error.

**Comment by ribsilva** [comment]
Install this: http://go.aptana.com/installer_nodejs_windows and then run the aptana installer again.

It worked for me

**Comment by millsy123** [comment]
Thanks **ribsilva** it worked

**Comment by BrianEARTH** [comment]
Thanks @ribsilva it worked perfectly ;)

**Comment by ronanjordan** [comment]
thanks ribsilva  it worked

**Comment by KFGL** [comment]
Your link is broken, and I do not know how to fix this. Also, why is Aptana Studio dependent on such a very low version of Node, and not allow any higher versions? I have the LTS version of Node, which happens to be v6.10.3.

**Comment by DarkPatriarch** [comment]
I finally got Aptana to install.  I used the .msi file on the following url to load the recommended version of Node.

https://nodejs.org/download/release/v0.10.48/

**Comment by KFGL** [comment]
Yeah, but it would be nice to be able to use a higher version, as I use nodejs for other purposes, and would prefer to only have the LTS version on my computer. Is there some way that this can be fixed?
